tests/html_basics/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -52,3 +52,55 @@ exports[`html-fragment.html 1`] = `
 <textarea />
 
 `;
+
+exports[`more-html.html 1`] = `
+<html>
+<body>
+    <a href="#">Anchor</a>
+    <div hidden class="foo" id=bar></div>
+</body>
+</html>
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+
+<html>
+
+<head />
+
+
+<body>
+  <a href="#">Anchor</a>
+  <div hidden
+    class="foo"
+    id="bar" />
+
+</body>
+</html>
+`;
```

Comment:
The html parser/output seems to be completely broken :p

Reply:
Yeah it's not great. I could fix (it's very similar to handlebars) it but I wanted to scope my PR around just getting handlebars working because it's already so much code.

Reply:
Yeah, don’t worry about the html printer